appveyor.yml

```diff
@@ -11,7 +11,7 @@ platform:
 environment:
   matrix:
     - TOOLCHAIN_VERSION: 14.0
-      RUST: 1.26.0
+      RUST: 1.28.0
```

Comment:
This should be done as a separate PR and we should be doing Rust 1.29.1.

Reply:
Is there a reason this isn't just "stable" instead of an explicit version number?